index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <div id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="">
+      <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+        increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </div>
+    <section id="tribute-info">
+      <h3 class="tribute-name">Here's a time line of Dr. Borlaug's life:</h3>
+      <ul class="tribute-ul">
+        <li class="tribute-li">
+          <span class="b">1914</span> - Born in Cresco, Iowa
+        </li>
+        <li class="tribute-li">
+          <span class="b">1933</span> - Leaves his family's farm to attend the University of Minnesota, thanks to a
+          Depression era program known as the "National Youth Administration"
+        </li>
+        <li class="tribute-li">
+          <span class="b">1935</span> - Has to stop school and save up more money. Works in the Civilian Conservation
+          Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+        </li>
+        <li class="tribute-li">
+          <span class="b">1937</span> - Finishes university and takes a job in the US Forestry Service
+        </li>
+        <li class="tribute-li">
+          <span class="b">1938</span> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts.
+          Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding
+          pest-resistent plants.
+        </li>
+      </ul>
+    </section>
+    <div class="footer">
+      <h2>
+        If you have time, you should read more about this incredible human being on his <a id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>.
+      </h2>
```

Comment:
Please fix the linter error.
Link 'rel' attribute should include 'noopener' or 'noreferrer'.